ci/jenkins/Jenkinsfile_py3-master_gpu_doc

```diff
@@ -67,10 +67,10 @@ core_logic: {
               if (env.BRANCH_NAME.startsWith('PR-')){
                 sh """
                 set +e
-                conda activate ./conda/cpu/py3
+                conda activate ./conda/cpu/py3-master
 
                 python3 ci/batch/submit-job.py --region us-east-1 --wait \
-                  --timeout 1800 --saved-output ./docs/examples --conda-env docker/py3 \
+                  --timeout 1800 --saved-output ./docs/examples --conda-env docker/py3-master \
```

Comment:
`docker/py3-master` does not exist. You need to update the `docker/py3` file instead.
We only run one job for AWS Batch. Not one for master and stable.